locksmith/src/utils/pricing.ts

```diff
@@ -1,4 +1,14 @@
 import { networks } from '@unlock-protocol/networks'
+import { Web3Service, getErc20Decimals } from '@unlock-protocol/unlock-js'
+import { ethers } from 'ethers'
+import logger from '../logger'
+import GasPrice from './gasPrice'
+
+// Stripe's fee is 30 cents plus 2.9% of the transaction.
+const baseStripeFee = 30
+const stripePercentage = 0.029
+export const GAS_COST = 200000 // hardcoded : TODO get better estimate, based on actual execution
+export const GAS_COST_TO_GRANT = 250000
```

Comment:
We already define this somewhere else, please re-use otherwise it is going to be a pain to maintain. (mauyve mobve to constants if that's easier, same for vars above)